docs/examples/loaders/SVGLoader.html

```diff
@@ -11,9 +11,34 @@
 
 		<h1>[name]</h1>
 
-		<div class="desc">A loader for loading an <em>.svg</em> resource.</div>
+		<div class="desc">A loader for loading a <em>.svg</em> resource.<br >
+		<a href="https://en.wikipedia.org/wiki/Scalable_Vector_Graphics">Scalabe Vector Graphics</a> is an XML-based vector image format for two-dimensional graphics with support for interactivity and animation.			
+		</div>
+
+		<h2>Example</h2>
+		
+		<code>
 
+		// instantiate the loader
+		var loader = new THREE.SVGLoader();
+		
+		// load a SVG resource
+		loader.load(
+			//	resource URL
+			'data/svgSample.svg',
+			// called when the resource is loaded.
+			function(doc){
```

Comment:
In this code block, and the other code blocks in this PR, you can use http://zz85.github.io/mrdoobapproves/ to help you get the formatting correct.

Reply:
Thanks for the input. 👍 
Fixed it. 🙂

Reply:
Btw, that's an interesting tool. Much helpful in getting the project requirements out there with out too much documentation and feedback from the project maintainers 😎

Reply:
Many thanks for your contribution!